packages/discovery-provider/src/tasks/entity_manager/entity_manager.py

```diff
@@ -1041,6 +1078,55 @@ def fetch_existing_entities(session: Session, entities_to_fetch: EntitiesToFetch
             for _, dashboard_wallet_json in dashboard_wallets
         }
 
+    logger.info(f"asdf entities_to_fetch {entities_to_fetch}")
```

Comment:
are these meant to stay or just temporary

Reply:
this is my trademark